packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -628,7 +632,10 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
   /// This callback should be used to update the scroll behavior, if
   /// necessary, and then to call [updateGestureDetector] to update
   /// the gesture detectors accordingly.
-  Widget buildContent(BuildContext context);
+  Widget buildContent(BuildContext context) {
+    // TBD: if config.builder is null throw a sensible error.
```

Comment:
It should have been an assert. I think it took longer to write the TBD note to myself (which I promptly forgot about) than to just write the assert :-).